static/app/views/aiMonitoring/landing.tsx

```diff
@@ -61,18 +76,26 @@ export default function AiMonitoringPage() {
                         <DatePageFilter />
                       </PageFilterBar>
                     </ModuleLayout.Full>
-                    <ModuleLayout.Third>
-                      <TotalTokensUsedChart />
-                    </ModuleLayout.Third>
-                    <ModuleLayout.Third>
-                      <NumberOfPipelinesChart />
-                    </ModuleLayout.Third>
-                    <ModuleLayout.Third>
-                      <PipelineDurationChart />
-                    </ModuleLayout.Third>
-                    <ModuleLayout.Full>
-                      <PipelinesTable />
-                    </ModuleLayout.Full>
+                    {isOnboarding ? (
```

Comment:
Does this cause a flash while the data's loading, by any chance? The other modules explicitly hide onboarding while data is still loading

Reply:
No, it defaults to false - not sure why it would flash